xAPI-Data.md

```diff
@@ -447,8 +448,8 @@ The table below lists all properties of an Anonymous Group.
 
 <table border ="1">
 	<tr><th>Property</th><th>Type</th><th>Description</th><th>Required</th></tr>
-	<tr id="2.4.2.2.s2.table1.row1"><td>objectType</td><td>String</td><td>"Group". </td><td>Required</td></tr>
-	<tr id="2.4.2.2.s2.table1.row2"><td>name</td><td>String</td><td>Name of the group.</td><td>Optional</td></tr>
+	<tr id="2.4.2.2.s2.table1.row1"><td>objectType</td><td>String</td><td>`Group`. </td><td>Required</td></tr>
```

Comment:
Yes, I realized this wasn't about partway through my commits and would need a touch up later.  Thanks for guidance on this that you give in the following comments